### DISCUSSION TITLE
Problem of ZEPETO Studio Unity file folder 3.1.11.1

### DISCUSSION THREAD

**Discussion by Lumis015** [naverz/zepeto-studio-kor#322] [5. Q&A]
Is anybody got meet this problem?or only me?
Now I use Unity 2020.3.9, and FileFolder 3.1.11.1 (2022.01.27 updated)
When I export my FBX to ZPT type file, it will become larger, 5MB like that, so can't upload to shop(maximum is 1MB) it make me so confuse .

because the old version didn't have this problem before, is anybody can answer me?Thanks for much!

**Comment by Zepeto-TechEvangelist** [comment]
Hello, please check if the problem still occurs in the latest version.
Thank you.
